docs/admin_impersonation.md

```
@@ -0,0 +1,85 @@
+- Rake task: `lib/tasks/user.rake`
+- Route: `admin_create_impersonation_url`
+
+Command:
+
+```bash
+bundle exec rake user:generate_impersonation_url[email=user@example.com]
+```
+
+This prints a URL like:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)